src/GraphQL/Resolver.hs

```diff
@@ -105,11 +113,11 @@ ok :: GValue.Value -> Result GValue.Value
 ok = pure
 
 
--- TODO instead of SelectionSet we want something like
--- NormalizedSelectionSet which has query fragments etc. resolved.
+-- TODO: It's not crystal clear that "SelectionSet" should be the entry point.
```

Comment:
Done.

I think the issue w/ `Result` is that it's not yet quite the right kind of abstraction. I have Ideas.